app/modules/individuals/resources.py

```diff
@@ -482,3 +481,37 @@ def post(self, individual):
                 code=500,
             )
         return merge
+
+
+@api.route('/<uuid:individual_guid>/featured_image', doc=False)
```

Comment:
Why doc=False for these?

Reply:
absolutely no idea, must have copied from somewhere and not looked. 
What does that do?